src/EnergyPlus/Furnaces.hh

```diff
@@ -676,6 +678,18 @@ namespace Furnaces {
 		Real64 & PartLoadRatio // coil part-load ratio
 	);
 
+	void
+	SetMinOATCompressor(
```

Comment:
I am checking min OAT in the coil objects. But I also wanted to see if I could eliminate unnecessary calculations when I know the coil would be off (e.g., if off return, or something like that). Turns out it was harder than I thought (for example I thought I could set the load = 0 in Init and be done but that also turns off the supp heater). There are existing places in the parent that mine that data and do the check so this replicates that method.